docs/cumulusci_flow.rst

```diff
@@ -0,0 +1,273 @@
+CumulusCI Flow
+==============
+CumulusCI Flow is the branching model and process we use for development of our products at Salesforce.org.
```

Comment:
Maybe add something to explain its overall goal, like "It's the way we arrange our work to make sure that changes don't get added to the main branch of the project until they have been adequately tested, and that changes added to main get integrated into other branches in a timely fashion."